src/%IPM/ResourceProcessor/Abstract.cls

```diff
@@ -225,35 +225,46 @@ Method OnGetUniqueName(Output pUniqueName)
 {
 }
 
-Method GetSourceControlInfo(Output pInfo As %IPM.PackageManager.Developer.Extension.SourceControl.ResourceInfo) As %Status
+Method GetSourceControlInfo(Output pInfo As %IPM.ExtensionBase.SourceControl.ResourceInfo) As %Status
 {
-	Set pInfo = ##class(%IPM.PackageManager.Developer.Extension.SourceControl.ResourceInfo).%New()
+	Set pInfo = ##class(%IPM.ExtensionBase.SourceControl.ResourceInfo).%New()
 	Quit $$$OK
 }
 
-/// Evaluates an expression in an attribute
+
+/// Evaluates an expression in a provided string. <br />
+/// These special expressions are case-insensitive. <br />
 /// Current valid expressions:
-/// {$namespace} - the current namespace
-/// {$mgrdir} - the instance's mgr directory
-/// {$cspdir} - the instance's root CSP directory
-/// {$root} - the resource's module's root directory
-/// {name} - the name of the resource
-/// These special expressions are case-insensitive.
-Method %Evaluate(pAttrValue) As %String [ Internal ]
+/// <ul>
+/// 	<li>{$root} - the resource's module's root directory </li>
+/// 	<li>{name} - the name of the resource </li>
+/// </ul>
+/// In addition to these, look at expressions supported by %EvaluateSystemExpression
+/// in class <class>%IPM.Utils.Module</class>.
+Method %Evaluate(pAttrValue As %String) As %String [ Internal ]
```

Comment:
needs to be restored what was here before

Reply:
Hi Dmitry! The %Evaluate method will call %EvaluateSystemExpression, which includes the current functionality of the old version of %Evaluate, so I don't see a very strong reason of reverting back to the old version of %Evaluate(), since all the existing calls to %Evaluate() will have the preserved functionality. Besides, it is an internal method, so I don't see a strong reason to have to revert it. You can review the %EvaluateSystemExpression() method to see if there's any missing features from the old version of %Evaluate(). Thank you!